inc/network/includes-network.php

```diff
@@ -496,33 +496,9 @@ function network_step2( $errors = false ) {
 		</label></p>
 		<textarea id="network-wpconfig-rules" class="code" readonly="readonly" cols="100" rows="31" aria-describedby="network-wpconfig-rules-description">
 <?php ob_start(); ?>
-if ( !empty( $_ENV['PANTHEON_ENVIRONMENT'] )) {
-    $site_name = $_ENV['PANTHEON_SITE_NAME'];
-    // Override $hostname value as needed.
-    switch ( $_ENV['PANTHEON_ENVIRONMENT'] ) {
-        case 'live':
-            $hostname = $_SERVER['HTTP_HOST'];
-            break;
-        case 'test':
-            $hostname = 'test-' . $site_name . '.pantheonsite.io';
-            break;
-        case 'dev':
-            $hostname = 'dev-' . $site_name . '.pantheonsite.io';
-            break;
-        case 'lando':
-            $hostname = $site_name . '.lndo.site';
-            break;
-        default:
-            $hostname = $_ENV['PANTHEON_ENVIRONMENT'] . '-' . $site_name . '.pantheonsite.io';
-            break;
-    }
-} else {
-    // Override with a default hostname.
-    $hostname = '<?php echo $hostname ?>';
-}
 define( 'MULTISITE', true );
 define( 'SUBDOMAIN_INSTALL', <?php echo $subdomain_install ? 'true' : 'false'; ?> );
-define( 'DOMAIN_CURRENT_SITE', $hostname );
+define( 'DOMAIN_CURRENT_SITE', $_SERVER['HTTP_HOST'] );
```

Comment:
Have you tested with Lando to confirm dropping the hard-coded hostname doesn't break anything locally?

Reply:
Tested with Lando and found issues:

1. Getting a PHP Notice when running `lando wp` cli commands: `PHP Notice:  Undefined index: HTTP_HOST in phar:///usr/local/bin/wp/vendor/wp-cli/wp-cli/php/WP_CLI/Runner.php(1296) : eval()'d code on line 90`
2. When pulling a WPMS site, users need to do a `lando wp search-replace` with `--all-tables` flag to include the `wp_blogs` & `wp_site` entries to be updated

Updated my PR to to fix the 1

For the 2 error, we might need to add a more verbose error to the generic error `Error establishing a database connection` to make sure that all the URLs are replaced accordingly